### DISCUSSION TITLE
[RELEASE] Alleyways

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#279] [Map Releases]
# Info about the Map

This is an extension/sequel to the map [Alleyway](https://github.com/nzp-team/nzportable/discussions/173)

It contains rather tight Alleyways and Streets which you need to fend off the hordes of undead, and it's generally more on the semi difficult side.

The main meta of [Alleyway](https://github.com/nzp-team/nzportable/discussions/173) still applies here, although now the mystery box moves around from time to time, so you may have to step into the new areas of this small chunk of a city.

This most likely runs **pretty alright** on the _PSP_ as I have tested it near the end of Development.

The new area runs around 30 Fps, while the original area stays near the 60 Fps area.

# Preview
![fte-20230327225107-0](https://user-images.githubusercontent.com/104042185/228068633-a5031da1-f0e4-40a8-9b09-655cfbb55433.png)
![fte-20230329144756-0](https://user-images.githubusercontent.com/104042185/228540060-e371bab2-42b9-4b83-bab4-a16e89c7c99d.png)
![fte-20230227172732-0](https://user-images.githubusercontent.com/104042185/228068626-38e30977-44fd-419e-81dc-c58cd2c51832.png)

# Update Log

Please use the download button down below to download the latest version!

- V1: [Alleyway](https://github.com/nzp-team/nzportable/files/10611034/Alleyway.zip) Made the Map.

- V2: [Alleyways](https://github.com/nzp-team/nzportable/files/11101793/Alleyways.zip) Expanded Alleyway and made the zombies walk better.
- V2.1: [Alleyways](https://github.com/nzp-team/nzportable/files/13645172/Alleyways.zip) removed Perk Jingles and removed the .map and wad into a seperate Zip [Source](https://github.com/nzp-team/nzportable/files/13645209/Uncompiled.Map.with.WAD.Unneeded.zip). This change was necessary to recieve the Meets Install Standard Label.

# [Download](https://github.com/nzp-team/nzportable/files/13645172/Alleyways.zip)

**Comment by Paindeluxe** [comment]
A month-long hiatus but now a new map will follow soon should be expected to come out in 2–4 days.

**Comment by Paindeluxe** [comment]
I might've forgotten to calculate the visleafs well it's fixed now.

**Comment by MotoLegacy** [comment]
Hey there, to be eligible for the "Meets Install Standard" label, please separate the map source files into a separate download link, appropriately named "Source" or something, instead of "DOWNLOAD".

**Reply by Paindeluxe** [reply]
This should be fixed now and the copyrighted Perk Jingles have been removed from the newest version.

**Reply by MotoLegacy** [reply]
Just to be clear -- copyrighted material being removed isn't a requirement for the label. The label is to embrace a consistent standard for installation, not for inclusion in the game, so there's no worrying about license compatibilities from me.
